src/components/MainClaim.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+export default function MainClaim() {
+  return (
+    <div className='text-main-mobile lg:text-main px-10 font-mono font-extrabold text-white lg:px-28'>
+      <a className='text-white/50'> UNLOCK YOUR </a> POTENTIAL
```

Comment:
why `a` as a tag?

Reply:
Well, good question 😆 I think I wanted to use` <p> `here. But probably `<span>` makes more sense in this case. I'll change it in all places 👍